### PR TITLE
Update docs to use v12 as default version

### PR DESCRIPTION
**Pull request by alexfornuto**
Do not merge until https://github.com/gravitational/teleport/pull/21317 is merged.

### CODE REVIEW

File: config.json
Comment (r0mant) [inline config.json:53]: @alexfornuto I think you need to move `latest: true` here, right?

**Review by r0mant — APPROVED** [review]
@alexfornuto Thanks for preparing this. Let's hold off on merging this until 12 is fully published please (not just until https://github.com/gravitational/teleport/pull/21317 merges). I can merge this myself when it's time.

File: config.json
Comment (alexfornuto) [reply config.json:53]: Yup, and add Deprecated to v9. Thanks!

**Review by r0mant — APPROVED** [review]

**Pull request merged**
